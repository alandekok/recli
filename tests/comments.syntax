test comments

# here too
and here

# why not 
now
