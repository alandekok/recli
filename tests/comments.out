# comments get ignored if we
test comments
# why don't we test here
and here
# what do we do
now  # ?
# how about
now # ??? or maybe we
test comments #
